app/styles/app.scss

```diff
@@ -187,3 +187,12 @@ button .material-icons {
         }
     }
 }
+
+
+
+.sticky-nav{
+    width: 100%;
+    position: fixed !important;
```

Comment:
.__layer-wrapper__06257 .layer-wrapper {
    position: relative;
    min-height: 55px;
}
it wasnt overriding the wrappers postion without it